Translation corrections (#273)
07th-mod/higurashi-console-arcs: Update/yoig_012.txt

//r大事なのは、結論じゃない。建前じゃない。カッコ良さじゃない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "大事なのは、結論じゃない。建前じゃない。カッコ良さじゃない。",

> 			NULL, "The important thing isn't the end result. Or the what excuses you make. Or how cool you look.", Line_Normal);

Reviewer: ```suggestion
			NULL, "The important thing isn't the end result. Or what excuses you make. Or how cool you look.", Line_Normal);
```

	ClearMessage();

//r大事だと思ったものに、本気でぶつかっていくことなんだ…！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "大事だと思ったものに、本気でぶつかっていくことなんだ…！",

> 			NULL, "What's truly important... is to face that which you value head on...!", Line_Normal);

Reviewer: ```suggestion
			NULL, "What's truly important... is to face the things you value head on...!", Line_Normal);
```
